Mathlib/MeasureTheory/Integral/IntegrableOn.lean

```diff
@@ -543,6 +543,24 @@ theorem integrable_add_of_disjoint {f g : α → E} (h : Disjoint (support f) (s
   · rw [← indicator_add_eq_right h]; exact hfg.indicator hg.measurableSet_support
 #align measure_theory.integrable_add_of_disjoint MeasureTheory.integrable_add_of_disjoint
 
+/-- If a function converges along a filter to a limit `a`, is integrable along this filter, and
+all elements of the filter have infinite measure, then the limit has to vanish. -/
+lemma IntegrableAtFilter.eq_zero_of_tendsto
+    (h : IntegrableAtFilter f l μ) (h' : ∀ s ∈ l, μ s = ∞) {a : E}
+    (hf : Tendsto f l (𝓝 a)) : a = 0 := by
+  by_contra H
+  obtain ⟨ε, εpos, hε⟩ : ∃ (ε : ℝ), 0 < ε ∧ ε < ‖a‖ := by
+    have: 0 < ‖a‖ := norm_pos_iff'.mpr H
+    exact ⟨‖a‖/2, half_pos this, div_two_lt_of_pos this⟩
```

Comment:
Can be shortened a bit with [`exists_between`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Order/Basic.html#exists_between).